apps/studio/.env.local

```diff
@@ -12,6 +12,8 @@ SANITY_STUDIO_PAGE_BUILDER_DEMO_PREVIEW_URL=""
 SANITY_STUDIO_REMIX_PREVIEW_URL=""
 SANITY_STUDIO_SVELTE_PREVIEW_URL=""
 
+SANITY_STUDIO_PRESENTATION_DEBUG_TELEMETRY=true
```

Comment:
You'll need to also add this to `apps/studio/turbo.json` I think

Reply:
Oh cool! It works without but want to be sure. Have added :)